woocommerce/wc-admin
# Added paystack to payment methods for african countries

Fixes #6538 

Add's the **Paystack** payment option for some african countries (ZA, GH, NG), and move the **Payfast** method up above Paypal.

### Accessibility




### Screenshots

<img width="1520" alt="Screen Shot 2021-03-12 at 1 25 35 PM" src="https://user-images.githubusercontent.com/2240960/110975698-750a1600-8336-11eb-90ba-8682f7a17055.png">

![paystack-provider](https://user-images.githubusercontent.com/2240960/110977062-0f1e8e00-8338-11eb-877f-38be13844ee1.gif)

### Detailed test instructions:

1. On a fresh Woo store install, go to the store setup wizard `/wp-admin/admin.php?page=wc-admin&path=%2Fsetup-wizard`
2. Set up your store with **South Africa - Eastern Cape** as its country, and proceed
3. Select **Fashion, apparel, and accessories** as the industry, and finish the rest of the onboarding flow.
4. Once finished it will redirect to the home screen, click on the **Choose payment methods** task
5. Both **Paystack** and **Payfast** should be listed above **Paypal**
6. Click **Set up** on the Paystack payment option
7. It should successfully finish the **Install** step, and ask for a public and secret key.
8. Check if the **Paystack account** link, directs you to the **dashboard.paystack.com/#/settings/developer**
9. Set `public_key` for public key, and `secret_key` for secret key and click **Proceed**
10. It should redirect you to the payment list and the **Set up** button should be gone, and replaced by an enabled toggle button.
11. You should be able to successfully toggle paystack from on to off and back. Leave it selected for now.
12. Go to **WooCommerce > Settings > Payments**, **Paystack** should be selected.
13. Click **Manage**, the secret and public key's should match what you entered in step 9.
